mdata/cache/accnt/stats.go

```diff
@@ -33,6 +33,9 @@ var (
 
 	cacheSizeMax          = stats.NewGauge64("cache.size.max")
 	cacheSizeUsed         = stats.NewGauge64("cache.size.used")
+	cacheOverheadChunk    = stats.NewGauge64("cache.overhead.chunk")
```

Comment:
it would be helpful to have a comment describing what each of those new metrics measure.